cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/CassandraDialect.java

```diff
@@ -534,6 +534,12 @@ public DuplicateInsertPreventionStrategy getDuplicateInsertPreventionStrategy(En
 	}
 
 	@Override
+	public int executeBackendUpdateQuery(BackendQuery<String> query, QueryParameters queryParameters) {
```

Comment:
Which base class do you mean? CassandraDialect does not extend a class, it implements two interfaces.